src/button.tsx

```diff
@@ -81,7 +81,7 @@ const renderButtonElement = ({
 
 const ButtonLinkChevron: React.FC = () => {
     return (
-        <svg width="8" height="20" viewBox="0 0 8 20">
+        <svg width="0.5em" height="1.25em" viewBox="0 0 8 20">
```

Comment:
please use `pxToRem()` function

Reply:
Discussed offline. Using `rem` is not correct. We want the chevron to adapt to the button text's font size, and this size changes between devices (14px in mobile, 16 in desktop). For this reason, it's better to leave it with `em`.